samples/cpp/resize_pil.cpp

```diff
@@ -0,0 +1,32 @@
+#include <opencv2/opencv.hpp>
+//Include file for every supported OpenCV function
+
+using namespace cv;
+
+int main(int argc, char** argv)
+{
+  Mat image;
+  image = imread("d:\\mask.bmp", IMREAD_GRAYSCALE);  // Read the file
```

Comment:
I think it is better to use synthetic image, maybe gradient?

Reply:
Yes, I've added  tests with gradient.png file

Reply:
@mshabunin, I've added test with gradient.png but my tests haven't passed due to absence of this file in test folder. Here is my code:

_const string root = cvtest::TS::ptr()->get_data_path();
const string filename = root + "gradient.png";
Mat img;
ASSERT_NO_THROW(img = imread(filename, IMREAD_GRAYSCALE));
ASSERT_FALSE(img.empty());_ 

and the error is:
_Value of: img.empty()
  Actual: true
Expected: false_

Should I add tests with another image or it is possible to add gradient.png in test data folder as well?

Reply:
I meant that you can generate source image on the fly, like [here](https://github.com/opencv/opencv/issues/9096#issuecomment-333673380).

Reply:
OK, I've done it in test_resize_pil.cpp